llmfoundry/models/hf/hf_causal_lm.py

```diff
@@ -131,6 +132,7 @@ def __init__(
             eval_metrics=eval_metrics,
             init_device=init_device,
             peft_config=peft_config_object,
+            should_save_peft_only=should_save_peft_only,
```

Comment:
lol we didn't pass this flag in before? funny cuz it's on composer's HuggingFaceModel class

Reply:
didn't pass cause assumed everyone wanted just peft checkpoints